### PR TITLE
Cards implemented to spec

### PR DESCRIPTION
**Pull request by Hoobachube**
Ideally I would rather use pictures instead of colours as the card backgrounds but unsure where i can get un-copyrighted jpegs.  Also I cannot seem to link back to the other pages once I have clicked on the individual apps.  I have tried every path I can think of.

### CODE REVIEW

File: apps.html [hunk 1]
```diff
@@ -35,9 +36,51 @@
...
+</div>
+  <ul>
+    <li>
+      <a href="apps/sudoku.html">sudoku
+      </a>
+    </li>
+    <li>
+      <a href="apps/connect4.html">connect4</a>
+    </li>
+  </ul>
```
Comment (alexw1983) [inline apps.html]: You can delete this it was just for demo purposes

File: apps.html [hunk 2]
```diff
@@ -35,9 +36,51 @@
...
+    <div class="card bg-success">
+    <div class="card-body text-center">
+      <a href="apps/favouritesgame.html">Favourites Game</a>
+    </div>
+  </div>
+  <div class="card bg-success">
+    <div class="card-body text-center">
+      <a href="apps/calculator.html">Calculator</a>
+    </div>
+  </div>
```
Comment (alexw1983) [inline apps.html]: Wrap all of this in a container

File: apps.html [hunk 3]
```diff
@@ -35,9 +36,51 @@
         <button class="btn btn-success my-2 my-sm-0" type="button">Search</button>
       </form>
     </div>
+    </div>
   </nav>
 
+  <div class="card-columns">
+    <div class="card bg-success">
+      <div class="card-body text-center">
+        <a href="apps/sudoku.html">sudoku</a>
```
Comment (alexw1983) [inline apps.html]: Capital S on sudoku

File: apps.html [hunk 4]
```diff
@@ -35,9 +36,51 @@
...
+      </div>
+    </div>
+    <div class="card bg-success">
+      <div class="card-body text-center">
+        <a href="apps/noughts&crosses.html">Noughts & Crosses</a>
+      </div>
+    </div>
+    <div class="card bg-success">
+      <div class="card-body text-center">
+        <a href="apps/1armedbandit.html">1 armed bandit</a>
```
Comment (alexw1983) [inline apps.html]: Capitals

**Review by alexw1983 — CHANGES_REQUESTED** [review]
A good effort. You are really coming along. I have left a few comments in the code for a bit of tidy up.

Instead of copyrighted images we can maybe use icons.

Also the blue text on a green background isn't very clear. Let's rework that

File: apps/1armedbandit.html [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+        <div class="container">
+          <a class="navbar-brand" href="index.html">Jack's App Chest</a>
+          <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navb">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+    
+          <div class="collapse navbar-collapse" id="navb">
+            <ul class="navbar-nav mr-auto">
+              <li class="nav-item">
+                <a class="nav-link" href="C:\Vs Code\jacks-app-chest\index.html">Home</a>
```
Comment (alexw1983) [inline apps/1armedbandit.html]: use ../index.html for the href same for all the others

File: apps.html (hunk 1 above)
Comment (Hoobachube) [reply apps.html]: done

File: apps.html (hunk 2 above)
Comment (Hoobachube) [reply apps.html]: done

File: apps.html (hunk 3 above)
Comment (Hoobachube) [reply apps.html]: done

File: apps.html (hunk 4 above)
Comment (Hoobachube) [reply apps.html]: done

File: apps/1armedbandit.html (hunk 5 above)
Comment (Hoobachube) [reply apps/1armedbandit.html]: done

**Review by alexw1983 — APPROVED** [review]

**Pull request merged**